### PR TITLE
1.10.2 development koa party

### PR DESCRIPTION
**Pull request by Corosauce**
- Village gen rate / range tweaks
- EIH rate tweak
- koa drum party every 3rd night
- Village schematic tweaks to improve koa navigation around it

### CODE REVIEW

File: src/main/java/net/tropicraft/core/common/entity/ai/EntityAIPartyTime.java [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+                    listDelays.add(3);
+                    listDelays.add(3);
+                    listDelays.add(6);
+
+                    lookupStateToSequence.put(index1++, listDelays);
+                    lookupStateToSequence.put(index1++, listDelays);
+                    lookupStateToSequence.put(index1++, listDelays);
+
+                    listDelays = new ArrayList<>();
+                    listDelays.add(9);
```
Comment (Cojomax99) [inline src/main/java/net/tropicraft/core/common/entity/ai/EntityAIPartyTime.java]: You could probably shorten this to List.addAll combined with Arrays.AsList. Something like: listDelays.addAll(Arrays.asList([9, 3, 3, 3, etc]));

Comment (tterrag1098) [reply src/main/java/net/tropicraft/core/common/entity/ai/EntityAIPartyTime.java]: or `Lists.newArrayList(...)`

**Review by Cojomax99 — APPROVED** [review]
